### DISCUSSION TITLE
zc300: Remove support for 'legacy' language files?

### DISCUSSION THREAD

**Discussion by lat9** [zencart/zencart#6729] [General]
Language 'array' files, e.g. `lang.english.php` were introduced in the zc158 release on October 16<sup>th</sup> 2022.  The 'conversion' to language arrays was in preparation for the change in PHP 8.0 whereby duplicate constants are now considered `PHP Warnings` instead of `PHP Notices`.

According to [PHP: Supported Versions](https://www.php.net/supported-versions.php), PHP versions prior to **8.2** are no longer actively supported.  As more and more web-hosts are dropping support for unsupported PHP versions &mdash; and especially those versions that are no longer receiving security support &mdash; is it time to drop support for those 'legacy' language files?

**Comment by torvista** [comment]
Yes.
